tests/unit/plugins/module_utils/test_ome.py

```diff
@@ -215,70 +225,250 @@ def test_get_device_id_from_service_tag_ome_case02(self, mocker, mock_response):
+                             ODATA_TYPE: DDEVICE_TYPE,
+                             "DeviceType": 1000,
+                             "Name": "SERVER",
+                             "Description": "Server Device"
+                         },
+                         {
+                             ODATA_TYPE: DDEVICE_TYPE,
+                             "DeviceType": 2000,
+                             "Name": "CHASSIS",
+                             "Description": "Chassis Device"
+                         }
+                     ]}
+        module_params = {'hostname': '192.168.0.1', 'username': 'username',
+                         'password': 'password', "port": 443}
+        mocker.patch(MODULE_UTIL_PATH + INVOKE_REQUEST,
+                     return_value=mock_response)
+        with RestOME(module_params, True) as obj:
+            ret = obj.strip_substr_dict(data_dict)
+        assert ret == {'value': [{'@odata.type': '#DeviceService.DeviceType', 'Description': 'Server Device', 'DeviceType': 1000, 'Name': 'SERVER'}, {
+            '@odata.type': '#DeviceService.DeviceType', 'Description': 'Chassis Device', 'DeviceType': 2000, 'Name': 'CHASSIS'}]}
+
+    def test_job_submission(self, mocker, mock_response):
+        mock_response.success = True
+        mock_response.status_code = 200
+        mock_response.json_data = {
+            'JobStatus': "Completed"
+        }
+        mocker.patch(MODULE_UTIL_PATH + INVOKE_REQUEST,
+                     return_value=mock_response)
+        module_params = {'hostname': '192.168.0.1', 'username': 'username',
+                         'password': 'password', "port": 443}
+        with RestOME(module_params, True) as obj:
+            ret = obj.job_submission(
+                "job_name", "job_desc", "targets", "params", "job_type")
+        assert ret.json_data == mock_response.json_data
+
+    def test_test_network_connection(self, mocker, mock_response):
+        mocker.patch(MODULE_UTIL_PATH + INVOKE_REQUEST,
+                     return_value=mock_response)
+        mock_response.success = True
+        mock_response.status_code = 200
+        mock_response.json_data = {
+            'JobStatus': "Completed"
+        }
+        mocker.patch(MODULE_UTIL_PATH + JOB_SUBMISSION,
+                     return_value=mock_response)
+        module_params = {'hostname': '192.168.0.1', 'username': 'username',
+                         'password': 'password', "port": 443}
+        with RestOME(module_params, True) as obj:
+            ret = obj.test_network_connection(
+                "share_address", "share_path", "share_type", "share_user", "share_password", "share_domain")
+        assert ret.json_data == mock_response.json_data
+
+        with RestOME(module_params, True) as obj:
+            ret = obj.test_network_connection(
+                "share_address", "share_path", "share_type")
+        assert ret.json_data == mock_response.json_data
+
+    # @pytest.mark.parametrize("ret_val", [
```

Comment:
pls, remove commented code

Reply:
updated to refactor the module params and conn obj to fixtures